src/index.css

```diff
@@ -120,7 +120,10 @@
      * fill visible viewport
      * https://developers.google.com/web/updates/2016/12/url-bar-resizing
      */
-    @apply h-full overflow-hidden;
+    @apply h-full;
+
+    /* prevent horizontal scrolling, e.g. when drawer is open */
+    @apply overflow-x-hidden;
```

Comment:
I don't notice this, what is it for?

Reply:
You'd have to test the branch as it was in be22dc8. We have to clip the overflow horizontally because we have a couple of animations which translate the UI horizontally:
- Opening the drawer slides it from left to right slightly, and the rest of the UI slides to the right
- The route activity slides away to the right when you go back to the route list
Without this you can scroll the whole page horizontally.